contracts/internal/EverlongPositions.sol

```diff
@@ -169,17 +174,61 @@ abstract contract EverlongPositions is EverlongBase, IEverlongPositions {
     // │ Position Closing (Internal)                             │
     // ╰─────────────────────────────────────────────────────────╯
 
+    /// @inheritdoc EverlongBase
+    function _increaseIdle(
+        uint256 _target
+    ) internal override returns (uint256 idle) {
+        // Obtain the current amount of idle held by Everlong and return if
+        // it is above the target.
+        idle = IERC20(_asset).balanceOf(address(this));
+        if (idle >= _target) return idle;
```

Comment:
We can update the style guide for this, but I prefer multi-line if-statements. The reason for this is that it makes it much easier to debug since you can add console statements _inside the if-statement_ to see what is happening.

```suggestion
        if (idle >= _target) {
            return idle;
        }
```